package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "kyt build --config kyt.config.production.js && sw-precache --config=sw-precache-config.js",
     "storybook": "start-storybook -p 6006 --dont-track",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test:e2e": "PROTOCOL=http HOST=0.0.0.0 PORT=8000 TEST_ENV=e2e kyt test"
```

Comment:
Related to the previous comment, this might work...

``
"test:e2e": "kyt test --config kyt.config.e2e.js"
``